adr/ADR-8.md

```diff
@@ -19,6 +19,7 @@
 | 4        | 2023-10-25 | Support compression                                 |
 | 5        | 2024-06-05 | Add KV management                                   |
 | 6        | 2024-06-05 | Add Keys listers with filters                       |
+| 7        | 2025-02-23 | Add Max Age limit Markers, remove non direct gets   |
```

Comment:
I wonder - maybe we could hint required server version in the metadata?

Reply:
2.11 already will add those right? 

For 2.12 we hope to make it do the API calls can say only create this stream on servers capable of level 1 etc

but I am not sure how we make this so we can detect 2.10 without calling api info or checking headers after create. 

How do you mean @Jarema ?

Reply:
I was just considering adding info to this table which server version introduced/enabled this feature :)

Reply:
doh, over thinking it :) yeah I guess we can add that

Reply:
have added a few there